bindings/python/multibody/joint/joints-variant.hpp

```diff
@@ -31,7 +33,23 @@ namespace pinocchio
       }
     };
 
-    struct exposer
```

Comment:
Please use the naming convention used in the rest of the projet.

Reply:
I am using DataExposer and ModelExposer now.